.travis.yml

```diff
@@ -1,24 +1,19 @@
 language: python
+python:
+  - "3.5"
 install:
   - pip install coveralls tox
 script:
   - tox
 env:  # generate list with: $ tox -l | xargs -I ITEM echo "  - TOXENV="ITEM
-  - TOXENV=py26-django14
-  - TOXENV=py26-django15
-  - TOXENV=py26-django16
-  - TOXENV=py27-django14
-  - TOXENV=py27-django15
-  - TOXENV=py27-django16
   - TOXENV=py27-django17
-  - TOXENV=py27-django18
-  - TOXENV=py32-django15
-  - TOXENV=py32-django16
   - TOXENV=py32-django17
-  - TOXENV=py32-django18
-  - TOXENV=py33-django15
-  - TOXENV=py33-django16
   - TOXENV=py33-django17
-  - TOXENV=py33-django18
   - TOXENV=py34-django17
+  - TOXENV=py33-django18
```

Comment:
@hobarrera Are you sure this is autogenerated code? According to the modified `tox.ini` there would be `py27-*` entries following after the last `py35-*`.

I don't want to be picky, but I feel we should care about consistency. Either we make this code here match the output of `tox -l` exactly (that's why there is the comment at the beginning of the `env` block after all), or at least we make the list beautiful in an obvious way.

Suggestion: Please change the comment on top of the `env` block to:
````bash
$ tox -l | sort | xargs -I ITEM echo "  - TOXENV="ITEM
```
(Note the additional, piped `sort` call.) Then generate the list again with that command, and paste it in here.

P.S.: I don't think it's easy to "automate" this in a sensible way, because we have 2 dimensions (python, django). If it is only one, it's fairly straightforward with Travis-CI. See [setup.cfg](https://github.com/Organice/django-organice/blob/master/setup.cfg#L17) and [.travis.yml](https://github.com/Organice/django-organice/blob/master/.travis.yml#L12) in django-organice, for example. -- If you can, though, do it. That's appreciated. I don't like crypographic shell scripting though. As we say in Python, *"There must be a better way!"*